modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/java/com/liferay/multi/factor/authentication/fido2/web/internal/checker/FIDO2BrowserSetupMFAChecker.java

```diff
@@ -191,6 +184,18 @@ public boolean isBrowserVerified(
 		return false;
 	}
 
+	public void removeCredentialEntry(long credentialEntryId) {
```

Comment:
we can move this implementation to the action directly since this does not need to be implemented on the checker